### DISCUSSION TITLE
Hydrogen project works locally but not on Vercel

### DISCUSSION THREAD

**Discussion by leiliu98** [vercel/community#966] [Help] [answered]
Hi,

I have done the steps running a local Hydrogen project (the demo snowboard project swapped in with my store's API token) and it worked perfectly. I then checked into Github and created a Vercel project and then the Vercel build also worked just fine. However, when accessing the URL, it shows “not found". (screenshot in following). It seems a very basic problem but I just can not figure out where to dig into it. Could anyone shed some light? The URL is at: [slimq-hydrogen.vercel.app](https://slimq-hydrogen.vercel.app/)

![image](https://user-images.githubusercontent.com/7281444/196709587-069094ec-f20c-492d-bd57-ddcfdea2dbf8.png)

**Comment by amyegan** [comment, marked as answer]
Hi @leiliu98! Do you have all of your [environment variables](https://vercel.com/docs/concepts/projects/environment-variables) added to your Project for the production environment? And are you using any of them in your Hydrogen config? This problem might be related to https://github.com/vercel/community/discussions/935
